apps/web/locales/en-US/release.json

```diff
@@ -47,7 +47,8 @@
   "Reserve Price": "Reserve Price",
   "Starting Bid": "Starting Bid",
   "Starting In": "Starting In",
-  "successConfirmation": "Thank you for your purchase. You are the proud owner of {{name}} and can view it in your collection.",
+  "successOwnershipConfirmation": "Congratulations, you are the proud owner of {{name}}!",
+  "successTransferConfirmation": "{{name}} has been transferred to your account and you can now view it in your collection.",
```

Comment:
Since claiming and transferring happens in two operations, I've improved up the language displayed for each.